pkgs/by-name/wl/wlogout/package.nix

```diff
@@ -64,6 +64,16 @@ stdenv.mkDerivation (finalAttrs: {
 
     substituteInPlace main.c \
       --replace "/etc/wlogout" "$out/etc/wlogout"
+
+    # aarch64 build (cross-compilation) fails with -Werror like this:
+    # aarch64-unknown-linux-gnu-gcc ... -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Werror -std=c11 -Wno-unused-parameter -Wno-unused-result -Wno-missing-braces -MD -MQ wlogout.p/main.c.o -MF wlogout.p/main.c.o.d -o wlogout.p/main.c.o -c ../main.c
+    # ../main.c: In function 'set_fullscreen':
+    # ../main.c:499:17: error: unused variable 'mon' [-Werror=unused-variable]
+    # 499 |     GdkMonitor *mon = gdk_display_get_monitor(gdk_display_get_default(), monitor);
+    #     |                 ^~~
+    # cc1: all warnings being treated as errors
+    substituteInPlace meson.build \
+      --replace-warn "werror=true" "werror=false"
```

Comment:
Maybe `env.NIX_CFLAGS_COMPILE= lib.optionalString <some cross situations> "-Werror=unused-variable"`?

Reply:
Thanks for finding that :)